.github/workflows/build.yml

```diff
@@ -68,6 +68,11 @@ jobs:
           "$GITHUB_SHA" \
           "$TARGET_NETWORK"
 
+          chmod +x .github/bin/inject-sentry-client-data.sh
+          .github/bin/inject-sentry-client-data.sh \
+          "$GITHUB_SHA" \
+          "$TARGET_NETWORK"
```

Comment:
How about summing each file containing client data for mixpanel and sentry into one file?

Reply:
good idea 👍